src/__swaps__/screens/Swap/components/ExchangeRateBubble.tsx

```diff
@@ -85,27 +85,27 @@ export const ExchangeRateBubble = () => {
 
       switch (rotatingIndex.value) {
         case 0: {
-          const formattedRate = valueBasedDecimalFormatter(
-            inputAssetPrice / outputAssetPrice,
-            outputAssetPrice,
-            'up',
-            -1,
-            isOutputAssetStablecoin,
-            false
-          );
+          const formattedRate = valueBasedDecimalFormatter({
+            amount: inputAssetPrice / outputAssetPrice,
+            usdTokenPrice: outputAssetPrice,
```

Comment:
Question that's not related to this PR but I just thought of. If it's possible to receive the token price in the user's desired selected currency, we should rename this to `nativeTokenPrice` or something more indicative that it isn't solely USD

Reply:
will take a note of this and adjust while I'm doing my changes! great callout!